scripts/gui/microbe_editor.mjs

```diff
@@ -229,42 +238,42 @@ function setRedo(enabled){
 }
 
 //! Sends organelle selection to the Game
-function onSelectNewOrganelle(organelle){
+function onSelectNewOrganelle(element){
```

Comment:
This still has the problem that this takes in the DOM element.

Reply:
i pass all the element, both DOM and organelle, element is for each variable of organelleSelectionElements

Reply:
I want this function to be as generic as possible. So this should only take in the name of the organelle.